plugins/inputs/nats_consumer/nats_consumer.go

```diff
@@ -46,6 +46,7 @@ type natsConsumer struct {
 	Username    string   `toml:"username"`
 	Password    string   `toml:"password"`
 	Credentials string   `toml:"credentials"`
+	JsSubjects  []string `toml:"js_subjects"`
```

Comment:
With the change above you should do
```suggestion
	JsSubjects  []string `toml:"jetstream_subjects"`
```